editor/js/Sidebar.Scene.js

```diff
@@ -376,29 +368,37 @@ function SidebarScene( editor ) {
 				backgroundType.setValue( "Color" );
 				backgroundColor.setHexValue( scene.background.getHex() );
 				backgroundTexture.setValue( null );
-				backgroundCubeTexture.setValue( null );
-				backgroundEquirectTexture.setValue( null );
+				backgroundEquirectangularTexture.setValue( null );
 
-			} else if ( scene.background.isTexture && ! scene.background.isPmremTexture ) {
+			} else if ( scene.background.isEquirectangularTexture ) {
```

Comment:
Where does `isEquirectangularTexture` comes from? Is this property added in a different commit?

Reply:
Hmm... Yeah, still figuring that out. I should comment that code to avoid confusion.

Reply:
Done. https://github.com/mrdoob/three.js/commit/a36f902f6f59de4d0c717bf3d4be9947667c7895